packages/webpack-bundler-runtime/src/constant.ts

```diff
@@ -1,2 +1,9 @@
 export const ENCODE_NAME_PREFIX = 'ENCODE_NAME_PREFIX';
-export const FEDERATION_SUPPORTED_TYPES = ['script'];
+export const FEDERATION_SUPPORTED_TYPES = [
+  'script',
+  'commonjs',
+  'commonjs2',
+  'commonjs-module',
+  'commonjs-static',
+  'node-commonjs',
```

Comment:
in fufure we should do "internal" as well? 

@jacob-ebey this would let you reference internal modules like "internal ./src/something" and get the module like webpack_require, as long as its bundled in somehow